STYLEGUIDE.md

```diff
@@ -32,8 +31,10 @@ Assume the reader is a busy non-native English speaker.
 ### Referring to the Product and Features
 
 - Sourcegraph: main product, prefer using this name unless you need to be more precise
-  - Sourcegraph Server: the form of our product that ships as the `sourcegraph/server` Docker image and runs on a single node
-  - Sourcegraph Data Center: the form of our product that runs on Kubernetes
+  - Sourcegraph OSS: the open source code for Sourcegraph. This is **not** the same as Sourcegraph Core.
+  - Sourcegraph Core: previously was called Sourcegraph Server, this build is the free tier of Sourcegraph built separately from Sourcegraph OSS.
```

Comment:
> previously was called Sourcegraph Server

This isn't _exactly_ right, since what was previously called "Server" really now maps to both "Core" and "Enterprise Starter" in the new model

Reply:
Should I remove this?

Reply:
Yeah, I'd just remove that, because it's not correct. If you prefer, you could replace it with something else that explains how the old "Server" is now both "Core" and "Enterprise Starter" but it's your call on whether that's necessary at all.